fpsim/empowerment.py

```diff
@@ -50,30 +53,58 @@ def get_empowerment_init_vals(ppl):
     # from 0 to 100 years old.
     n = len(ppl)
 
-    empwr_states = ['paid_employment', 'sexual_autonomy', 'decision_wages', 'decision_health']
-    empowerment = {empwr_state: np.zeros(n, dtype=fpd.person_defaults[empwr_state].dtype) for empwr_state in empwr_states}
+    empwr_states = ['paid_employment', 'sexual_autonomy', 'decision_wages', 'decision_health',
+                    'fertility_intent']
+
+    empwr_states = {empwr_state: np.zeros(n, dtype=fpd.person_defaults[empwr_state].dtype) for empwr_state in empwr_states}
+
+    # NOTE: maybe we should initialise it with 'no'?
```

Comment:
Exactly! I'll initialise to 'no' directly rather than to  `None`.

Similarly, in `defaults.py` I had set 'categorical_intent' to `None` for a newborn/new agent, but it's probably better to initialise to 'no'.

Reply:
On second thoughts, perhaps it'd may be more accurate to set the `categorical_intent` to "cannot" for new agents, and the metric `fertility_intent` to False (because they cannot have intent). 

A woman's category of intent is updated as she enters the 15-49 age group for which we have probabilities for each category ("yes", "no" and "cannot" ) from PMA data.